engine/src/main/java/io/camunda/zeebe/engine/processing/streamprocessor/StreamProcessorBuilder.java

```diff
@@ -77,8 +77,8 @@ public StreamProcessorBuilder eventApplierFactory(
     return this;
   }
 
-  public StreamProcessorBuilder replayMode(final ReplayMode replayMode) {
-    processingContext.replayMode(replayMode);
+  public StreamProcessorBuilder replayMode(final StreamProcessorMode streamProcessorMode) {
```

Comment:
Align the name of the method with the parameter name. 

We should change it to `streamProcessorMode()`.